packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -192,12 +192,12 @@ function FrameElement({ id }) {
   );
   const isLinkActive = isAnythingTransforming && !isLocked;
 
-  const getBox = useUnits(({ actions }) => actions.getBox);
+  const getBoxWithBorder = useUnits(({ actions }) => actions.getBoxWithBorder);
 
   useLayoutEffect(() => {
     setNodeForElement(id, elementRef.current);
   }, [id, setNodeForElement]);
-  const box = getBox(element);
+  const boxWithBorder = getBoxWithBorder(element);
```

Comment:
Lasso-selecting on the canvas and including the border of an element but only the border, does not select that element:
![border-lasso](https://user-images.githubusercontent.com/637548/181540844-bce98eeb-2d22-44d9-8810-d8a87a50fd6a.gif)

Reply:
@barklund This should be fixed now -- the lasso has been updated to use `useElementPolygon` instead, seems to work great and made things a bit simpler: https://github.com/GoogleForCreators/web-stories-wp/pull/11940/commits/43e0f5f46fa7a4571b3bd9c19d98c2ab5be74179

Reply:
![lasso](https://user-images.githubusercontent.com/3294597/181729477-65773cf2-77f0-446f-a27a-fad0e8dd5932.gif)